ert_shared/storage/http_server.py

```diff
@@ -126,17 +139,33 @@ def response_by_name(self, ensemble_id, response_name):
             response = api.get_response(ensemble_id, response_name, None)
             if response is None:
                 raise werkzeug_exc.NotFound()
+            response["alldata_ref"] = "Unicorn magic!"
```

Comment:
should this be here?

Reply:
Haha, that is my easter egg for reviewers. It just needs the key in the map, the value is never read. Don't even know if this is a good approach. But it works. Perhaps putting the value None is better?